custom_components/solarman/inverter_definitions/deye_hybrid.yaml

```diff
@@ -280,6 +280,42 @@ parameters:
         offset: 1000
         registers: [0x00B6]
 
+      - name: "Grid Charge Ampere"
+        platform: number
+        class: "current"
+        uom: "A"
+        scale: 1
+        rule: 1
+        registers: [0x00E6]
+        range:
+          min: 000
+          max: 999
+        icon: mdi:current-dc
+
+      - name: "Generator Charge Ampere"
+        platform: number
+        class: "current"
+        uom: "A"
+        scale: 1
+        rule: 1
+        registers: [0x00E3]
+        range:
+          min: 000
+          max: 999
+        icon: mdi:current-dc
+
+      - name: "Generator Force"
```

Comment:
This entity also exists in the SG04LP3 only as "Generator".

Reply:
deye_hybrid also has a "Generator" sensor. But this is different - that's the switch to force the generator. In your time of use you can set up with no charging from generator allowing for manual operation. So I think these should be separate - sensor for when Generator is on for whatever reason and this option to switch on/off manually (like I do)

<img width="1395" alt="image" src="https://github.com/user-attachments/assets/f314660e-7250-4c0a-8c92-0291b2c8df8e">

Reply:
This one also act as a force switch for the generator. 😉

Reply:
Ah, sorry I did read it too quickly I thought you was referring to the switch in the SG04LP3 profile which does the same thing. `0x00A6` shows the state of the relay, yes.

Reply:
Oh I see... Ok, do you want me to rename "Generator Force" to "Generator"?

Or maybe do it vice versa? I think "Generator Force" is a better name